core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -133,7 +133,11 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
     ) {}
```

Comment:
Did you check the WARN in the comment which say that there are duplicated extensions or blocks?